examples/pytorch/hilander/models/focal_loss.py

```diff
@@ -0,0 +1,66 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
```

Comment:
Do we still need focal_loss.py if it is not used now?

Reply:
I suggest we keep them since focal still perform better on some dataset like music. As long as this won't affect reproducing number, and considering we've already added them, we don't necessarily have to delete. Same thing for full graph training and inference, now all the numbers in the paper are get from sampled_training, but no harm to put the full graph code there.